.grunt-config/sass.js

```diff
@@ -79,11 +79,18 @@ const sass = {
 			},
 			{
 				expand: true,
-				cwd: 'assets/dev/scss/frontend',
+				cwd: 'assets/dev/scss/frontend/conditionals',
 				src: 'shapes.scss',
 				dest: 'assets/css',
 				ext: '.css',
 			},
+			{
+				expand: true,
+				cwd: 'assets/dev/scss/frontend/conditionals',
+				src: 'appleWebkit.scss',
+				dest: 'assets/css',
+				ext: '.css',
```

Comment:
@hein-obox Did you test this compatibility with RTL as well? In the past when this was compiled with the other grunt task, there would have been a separate css file created for `-rtl.scss` for example.
So i just want to make sure it still works fine in RTL.

Reply:
Compared in VS Code, and the styling is identical in `frontend-lite` and `frontend-lite-rtl`:

<img width="1220" alt="Screenshot 2024-07-19 at 12 19 50" src="https://github.com/user-attachments/assets/81d0e964-b0eb-4b9f-9124-e59ebe28b128">